src/codegen/utilities/interpreter_helpers.py

```diff
@@ -27,11 +27,13 @@
 ]
 
 
-def get_interpreter_functions(metadata):
+def get_interpreter_functions(metadata, exclude_grpc_only_functions=False):
     """Converts the scrapigen metadata into a list of functions."""
     all_functions = deepcopy(metadata["functions"])
     functions_metadata = []
     for function_name, function_data in all_functions.items():
+        if exclude_grpc_only_functions and is_grpc_only_function(function_data):
+            continue
```

Comment:
All of the read/write functions are marked grpc-only. What is the plan for supporting those? Add hand-written functions that redirect to https://github.com/ni/nidaqmx-python/blob/master/generated/nidaqmx/_task_modules/read_functions.py ?

Reply:
Brad, the current plan considered for read/write functions to remove the "grpc-only" mark so it will be considered for code generation. Since the read/write functions does not belong to a class name, it will be only considered for library interpreter generation & hence can be removed from the hand written wrapper `read_functions.py` as you have mentioned. Still out of these functions, two are not present in DAQmxAPISharp.json. Hence, those might have to be handwritten without the details metadata. 

This is the plan and the metadata is yet to be updated in grpc-device-scrapigen repo and tested. 

Meanwhile we have started the update of metadata for other function cases here - [in this PR](https://github.com/ni/grpc-device-scrapigen/pull/156/files). This PR covers update for standard Cdll & windll calls (Excluding read/write) and Event calls.